tests/teststun.nim

```diff
@@ -0,0 +1,82 @@
+{.used.}
```

Comment:
We probably need a copyright in these files.

Reply:
done